src/App.component.js

```diff
@@ -57,34 +61,35 @@ const mainContentTopPaddingPhone = 0;
 const phoneModeUpperThreshold = 768; // 768
 const tabletModeUpperThreshold = 1200; // 1200
 const modes = viewModes;
-export default React.createClass({
-    propTypes: {
-        d2: React.PropTypes.object,
-        WHOData: React.PropTypes.object,
-    },
 
-    childContextTypes: {
-        d2: React.PropTypes.object,
-    },
+class App extends React.Component {
 
-    mixins: [MuiThemeMixin],
+    static propTypes = {
```

Comment:
About the usage of static and the automatically bound methods, do we have a guideline on when we start using a spec? Because this is currently in stage 3, right (https://github.com/tc39/proposals#stage-3)? Potentially we could have to refactor this if the proposal doesn't make it. Just wondering.

Reply:
not sure if we have a written guideline for this, except that I've seen it in other of our code as well, for example d2-ui

Reply:
👍 I'll open a Jira issue for it